docs/DFAnumerics.rst

```diff
@@ -495,53 +495,91 @@ number of particles per kernel radius exceeds a given value :math:`n_{PPK}^{max}
-    \frac{\mathbf{du}_k}{\|\mathbf{du}_k\|}\\
-    = & - \frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k\| A_{Lat} \mathbf{du}_k
+    \mathbf{F_{viscosity}} = &- \frac{1}{2}\rho C_{Lat}\|\mathbf{du}\|^2 A_{Lat}
+    \frac{\mathbf{du}}{\|\mathbf{du}\|}\\
+    = & - \frac{1}{2}\rho C_{Lat}\|\mathbf{du}\| A_{Lat} \mathbf{du}
     \end{aligned}
 
-Where the velocity difference reads :math:`\mathbf{du}_k = \mathbf{u}_k - \mathbf{\bar{u}}`
+Where the velocity difference reads :math:`\mathbf{du} = \mathbf{u} - \mathbf{\bar{u}}`
 (:math:`\mathbf{\bar{u}}` is the mesh velocity interpolated at the particle position).
 :math:`C_{Lat}` is a coefficient that rules the viscous force. It would be the
 equivalent of :math:`C_{Drag}` in the case of the drag force. The :math:`C_{Lat}`
-is a numerical parameter that depends on the mesh size. Its value is set in the configuration (``subgridMixingFactor``) to 100
-by default (ToDo: and should be discussed and further tested).
+is a numerical parameter that depends on the mesh size. Its value is set to 100
+and should be discussed and further tested.
 
 Adding the viscous force
-~~~~~~~~~~~~~~~~~~~~~~~~~
+""""""""""""""""""""""""
 
 The viscous force is added implicitly:
 
 .. math::
   \begin{aligned}
-  \mathbf{F_k^{viscosity}} = &-\frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k^{old}\| A_{Lat}
-  \mathbf{du}_k^{new}\\
-  = &  -\frac{1}{2}\rho C_{Lat}\|\mathbf{u}_k^{old} - \mathbf{\bar{u}}^{old}\| A_{Lat}
-  (\mathbf{u}_k^{new} - \mathbf{\bar{u}}^{old})
+  \mathbf{F_{viscosity}} = &-\frac{1}{2}\rho C_{Lat}\|\mathbf{du}^{old}\| A_{Lat}
+  \mathbf{du}^{new}\\
+  = &  -\frac{1}{2}\rho C_{Lat}\|\mathbf{u}^{old} - \mathbf{\bar{u}}^{old}\| A_{Lat}
+  (\mathbf{u}^{new} - \mathbf{\bar{u}}^{old})
   \end{aligned}
-
 Updating the velocity is done in two steps. First adding the explcit term related to the
 mean mesh velocity and then the implicit term which leads to:
 
 .. math::
-  \mathbf{u}_k^{new} = \frac{\mathbf{u}_k^{old} - C_{vis}\mathbf{\bar{u}}^{old}}{1 + C_{vis}}
+  \mathbf{u}^{new} = \frac{\mathbf{u}^{old} - C_{vis}\mathbf{\bar{u}}^{old}}{1 + C_{vis}}
+
+With :math:`C_{vis} = \frac{1}{2}\rho C_{Lat}\|\mathbf{du}^{old}\| A_{Lat}\frac{dt}{m}`
+
+
+Ata Artificial viscosity
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-With :math:`C_{vis} = \frac{1}{2}\rho C_{Lat}\|\mathbf{du}_k^{old}\| A_{Lat}\frac{dt}{m}`
+An up-winding method based on Lax-Friedrichs scheme
+"""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Shallow Water Equations are well known for being hyperbolic transport equations.
+They have the particularity of carrying discontinuities or shocks which makes them
+numerically unstable.
+Thus a decentering in time allows to better capture the discontinuity.
+This can be done in the manner of the Lax-Friedrich scheme as described in :cite:`AtSo2005`.
+This decentering in time is formally the same as adding a viscous force.
+Transposed to the SPH method, this viscous force applied on a given particle :math:`k`
+can be expressed as follows:
```

Comment:
```suggestion
A decentering in time allows to better capture the discontinuities.
This can be done in the manner of the Lax-Friedrich scheme as described in :cite:`AtSo2005`, which is formally the same as adding a viscous force.
Implementing it for the SPH method, this viscous force applied on a given particle :math:`k`
can be expressed as follows:
```